capstone/fitness/src/main/java/com/google/sps/fit/JsonExercise.java

```diff
@@ -12,39 +12,48 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.google.sps.progress;
+package com.google.sps.fit;
 
-import com.google.sps.fit.*;
 import com.google.sps.fit.Exercise.SetType;
-import com.google.sps.util.*;
+import com.google.sps.progress.*;
 import java.util.*;
 
-/* Json version of progress marker. */
-public class JsonGoalStep {
+/* Json version of exercise. */
+public class JsonExercise {
 
   private final String name;
-  private final boolean complete;
```

Comment:
I removed the boolean because I wanted to make a generic exercise class that could be transformed into a json. In doing this, when we have session data, I can use this same JsonExercise class to send the session data to the display without any assumption that it's a GoalStep.